packages/base/src/features/patchPopup.ts

```diff
@@ -125,21 +193,16 @@ const patchOpen = (Popup: OpenUI5Popup) => {
 	};
 };
 
-const patchClosed = (Popup: OpenUI5Popup) => {
+const patchClosed = (Popup: OpenUI5PopupClass) => {
```

Comment:
if dialogs are destroyed instead of closed, the block layer remains with "popover=manual"
can be fixed in separate change/backlog